spec/features/user_views_posts_in_reverse_chronological_order_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "Posts", type: :feature do
+  scenario "Views posts in reverse chronological  order" do
+    visit "/posts/new"
```

Comment:
These 3 lines could be extracted to a helper method (e.g. add_post) so that it could be reused later.

Reply:
Thanks, changes made!